plugins/api-docs/index.js

```diff
@@ -100,6 +100,7 @@ async function copyApiDocs(manifest, processedMap) {
     console.log("[api-docs-plugin] No packages awaiting processing.");
     return;
   }
+	console.log("[api-docs-plugin] Processing packages:", packagesToProcess)
```

Comment:
question: is this needed? 

Indentation seems a bit different from the other lines 👀

Reply:
yes this is left over from debugging - good spot thanks